lib/factory-helper/timestamps.rb

```diff
@@ -0,0 +1,31 @@
+module FactoryHelper
+  class Timestamps < Base
+
+    class << self
+      def between(from= ::Time.at(0), to= ::Time.local(2050, 12, 31,  23,  23,  59, 999_999.999))
+        random_time(from, to)
+      end
+
+      def forward(from= ::Time.at(0), seconds= 60)
+        to = from + seconds
+        random_time(from, to)
+      end
+
+      def backward(from= ::Time.at(0), seconds= 60)
+        to = from - seconds
+        random_time(to, from)
+      end
+
+    private
+
+      def random_time(from, to)
+        diff_time = to - from
+        dif_in_nsec = (diff_time * 1_000_000_000).to_i
+        rand_in_nsec = FactoryHelper::Config.random.rand(dif_in_nsec-1) + 1 # avoids 0
+        rand_in_sec = rand_in_nsec.to_r / 1_000_000_000
+        from + rand_in_sec
+      end
```

Comment:
the problem here is that `- 1` will subtract 1 second and not 1 nanosecond.

But I think this will work:
`time_diff = (to - from).round(9)` #=> returns seconds with 9 digits 
Needs rounding because (`to - from = 0.0000000019999999...`) because of limited nr of bits in a float.
and
`FactoryHelper::Config.random.rand(time_diff - 0.000000001) + 0.000000001`

But even so, `.rand` will generate more digits than we need. (If we add `.floor`, it will not work with digits and if we add `.random(9)`, it might round to next nanosecond).
It looks simpler but might complicate the logic.


This `- 1) + 1` is there just to exclude the defined `from` and `to` times at the nanosecond level.
Between means all dates between the `from` and `to` and not between and included. So I considered just avoiding `from` and `to` from the end result (at nanosecond level only!).

E.g.
```
from = ::Time.local(2012, 05, 05,  05,  05,  30, 222_222.222)
to = ::Time.local(2012, 05, 05,  05,  05,  30, 222_222.224)
FactoryHelper::Timestamps.between(from, to) #=> always returns 2012-05-05 05:05:30 +0300
FactoryHelper::Timestamps.between(from, to).nsec #=> always returns 222222223
#as expected
# maybe we should add these to the tests too :)
```